cmd/partition.go

```diff
@@ -8,6 +8,7 @@ import (
 	"fmt"
 	"os"
 	"os/exec"
+  "strconv"
```

Comment:
Please run "go fmt ..." to fix up the formatting ... I'll add this as a pre-build step in the Makefile in a future update